README.md

```diff
@@ -277,26 +278,45 @@ those dependencies.
 [venv]: http://chaostoolkit.org/reference/usage/install/#create-a-virtual-environment
 
 ```console
-$ pip install -r requirements-dev.txt -r requirements.txt 
+$ make install-dev
```

Comment:
Does this also need to run: "$ make install" to install the requirements.txt file as well

Reply:
Nupe!

In the makefile, the definition of `install-dev` is:

```
.PHONY: install-dev
install-dev: install
	pip install -r requirements-dev.txt
	python setup.py develop
```

The second line: `install-dev: install` actually means run `make install` first, then run `install-dev`

Reply:
Perfect :)